pesa_migration/pesa_set_options.sql

```diff
@@ -6,6 +6,9 @@ update ##new_db##.or_options set option_value = 'migration' where (option_id = '
 update ##new_db##.or_options set option_value = 'n' where (option_id = '1426010286'); /* disable ethics module */
 update ##new_db##.or_options set option_value = 'y' where (option_id = '1426010298'); /* allow modification of reminder mails */
 update ##new_db##.or_options set option_value = 'y' where (option_id = '1426010305'); /* allow modification of sender mail */
+update ##new_db##.or_options set option_value = 10  where (option_id = '1426012101'); /* set max session duration hour default */
+update ##new_db##.or_options set option_value = 0   where (option_id = '1426012100'); /* set reserve participant default */
```

Comment:
what is this for?

Reply:
If you create a session you have to specify two numbers, the number of participants you require to make the experiment and a number of participants which will be a reserve. If you set the values to 1:0 (required:reserve) only on subject can enroll for a session and if the participant doesn't show up you can't run your experiment. By setting 1:1 two participants can enroll for the session (the first as a regular participant, the second as reserve). So even if one participant doesn't show up you will be able to run your experiment. If both show up you need to turn away one of the participants.

In this commit the setting is set to 1:0 (1 regular, 0 reserve) by default.